src/generator/www/js/main.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+
+  let $email =$("#email");
+  let $name  =$("#name"); 
+  let $url  =$("#url");
+
+  $("#btn").click(function() { 
+
+  var form = document.querySelector("form");
+  var fd = new FormData(form);
+
+  var files ={
+    "sessionfile":  $( "#sessionfile" )[0].files[0],
+    "speakerfile":$( "#speakerfile" )[0].files[0],
+    "trackfile":$( "#trackfile" )[0].files[0] ,
+    "locationfile":$( "#locationfile" )[0].files[0]
+  }
+  
+  for (var key in files) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711080098/issues/source?bid=3233857&fileBranchId=3344140#l19)